cumulusci/tasks/push/tasks.py

```diff
@@ -194,9 +195,14 @@ def _report_push_status(self, request_id):
 class SchedulePushOrgList(BaseSalesforcePushTask):
 
     task_options = {
+        "csv": {"description": "The CSV file to read.", "required": False},
+        "csv_field_name": {
+            "description": "The CSV field names to parse from the orgs file given. Please provide in string form.",
```

Comment:
You're using a plural here, when we only want one column, and we'll want to specify the default value. How about:
```suggestion
            "description": "The CSV field name that contains organization IDs. Defaults to 'OrganizationID'",
```